pkgs/top-level/python-packages.nix

```diff
@@ -6516,6 +6516,25 @@ in modules // {
 
   };
 
+  git-up = buildPythonPackage rec {
+    version = "1.4.0";
+    name = "git-up-${version}";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/g/git-up/${name}.zip";
+      sha256 = "0ydl6f4h593rd4h4xgjib7scj8jyvzq5sq6kf4145d0mcy7m5kmw";
+    };
+
+    buildInputs = with self; [ pytest ];
```

Comment:
does it need pytest? Looking at setup.py it uses nose for testing. Furthermore, looking at the archive it doesn't look like tests are included, in which case the tests should be disabled. When it builds it doesn't mean it is ready to be included ;-)